src/app/database/services/database.service/LRUFilesCacheManager.ts

```diff
@@ -0,0 +1,50 @@
+import { ICacheStorage, LRUCache } from './LRUCache';
+import databaseService, { DatabaseCollection, DriveItemBlobData } from '.';
+
+class LevelsBlobsCache implements ICacheStorage<DriveItemBlobData> {
```

Comment:
Nice abstraction here! Therefore we can switch it to use other cache implementations, especially for working with more restrictive browsers, etc.